client/reader/recent/style.scss

```diff
@@ -50,21 +86,30 @@
 			padding: 0;
 			width: 100%;
 
-			&__title-text,
-			&__site-name {
+			%text-shared {
 				white-space: nowrap;
 				text-overflow: ellipsis;
 				overflow: hidden;
 				max-width: 200px;
+
+				@media ( min-width: $break-wide ) {
+					max-width: 125px;
+				}
+
+				@media ( min-width: 1540px ) {
```

Comment:
I'm using a custom breakpoint here to prevent squishing the featured image at a specific screen size.